irods/test/collection_test.py

```diff
@@ -5,12 +5,14 @@
 import socket
 import shutil
 import unittest
+import time
 from irods.meta import iRODSMetaCollection
 from irods.exception import CollectionDoesNotExist
 from irods.models import Collection, DataObject
 import irods.test.helpers as helpers
 import irods.keywords as kw
 from six.moves import range
+from irods.test.helpers import my_function_name,unique_name
```

Comment:
space after comma.

there are a few places that need this treatment below as well.